default.nix

```diff
@@ -3,65 +3,128 @@
 ,
 }:
 let
-  pkgs = import inputs.nixpkgs {
+  pkgs = import inputs.nixpkgs-prev-stable {
     config = { };
     overlays = [ (import ./overlay.nix) ];
     inherit system;
   };
+  lib = pkgs.lib;
 
-  nix-dev = pkgs.stdenv.mkDerivation {
-    name = "nix-dev";
-    src = ./.;
-    nativeBuildInputs = with pkgs.python310.pkgs; [
-      linkify-it-py
-      myst-parser
-      sphinx
-      sphinx-book-theme
-      sphinx-copybutton
-      sphinx-design
-      sphinx-notfound-page
-      sphinx-sitemap
-    ];
-    buildPhase = ''
-      make html
-    '';
-    installPhase =
-      let
-        # Various versions of the Nix manuals, grep for (nix-manual)= to find where they are displayed
-        # FIXME: This requires human interaction to update! See ./CONTRIBUTING.md for details.
-        releases = {
-          latest = "2.19";
-          rolling = "2.18";
-          stable = "2.18";
-          prev-stable = "2.13";
-        };
-        inputName = version: pkgs.lib.strings.replaceStrings [ "." ] [ "-" ] version;
-        src = version: inputs."nix_${inputName version}";
-        manual = version: (import (src version)).default.doc;
-        copy = version: ''
-          cp -Rf ${manual version}/share/doc/nix/manual/* $out/manual/nix/${version}
-        '';
-        # add upstream page redirects of the form `<from> <to> <status>`, excluding comment lines and empty
-        redirects = version: ''
-          sed '/^#/d;/^$/d;s#^\(.*\) \(.*\) #/manual/nix/${version}\1 /manual/nix/${version}\2 #g' ${src version}/doc/manual/_redirects >> $out/_redirects
+  nix-dev =
+    let
+      # Various versions of the Nix manuals, grep for (nix-manual)= to find where they are displayed.
+      # XXX: With the current setup, the Nix manual hosted on nix.dev does not get updated automatically with new releases.
+      # The following manual steps are required:
```

Comment:
This would be better in `CONTRIBUTING.md` to replace the section that was previously there.

Reply:
Done